### PR TITLE
Add Arnaud to the team section

### PR DESCRIPTION
**Pull request by aclaudel**

### CODE REVIEW

File: src/_data/team.yml [hunk 1]
```diff
@@ -972,3 +972,15 @@
...
   email: etienne.mustow@codurance.com
   office: london
+
+- id: arnaud_claudel
+  name: Arnaud Claudel
+  image: /assets/custom/img/team/arnaud_claudel.jpg
+  role: Apprentice
+  fullDescription:
+    <p>Arnaud started to work in a big IT company but quickly got tired of the lack of quality in his projects.</p>
+    <p>He then tried to find new ways of working and met a lot of passioned people accross meetups.</p>
```
Comment (keith-smale) [inline src/_data/team.yml]: Just need to correct the spelling of 'accross' to 'across'.

**Review by keith-smale — CHANGES_REQUESTED** [review]
Awesome stuff. Just need to correct a spelling error.

Thank you for getting it done so quick.

File: src/_data/team.yml (hunk 1 above)
Comment (aclaudel) [reply src/_data/team.yml]: Thanks, I changed it

**Review by keith-smale — APPROVED** [review]

**Comment by keith-smale** [comment]
This is all good now but as someone else was added to the site since you made changes there are now conflicts in the team.yml. 
Can you resolve these and then we can get it merged.

**Comment by aclaudel** [comment]
I've changed it. It looks good on the preview.

**Pull request merged**